maui/src/SegmentedControl/Views/SegmentItemView.cs

```diff
@@ -875,7 +875,10 @@ void ITouchListener.OnTouch(PointerEventArgs e)
 		{
 			if (e.Action == PointerActions.Pressed)
 			{
-				_effectsView?.ApplyEffects();
+				if (itemInfo?.SegmentTemplate is null)
```

Comment:
It would be nice if there was also a property to control this behavior to support having a template and applying the effects.

Reply:
Hi @powerdude,  
Thank you for your suggestion! We have considered this as a [feature request](https://github.com/syncfusion/maui-toolkit/issues/103).